Truck.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax;
+public class Truck extends Machine {
+    @java.lang.Override
+    public void doWork() {
+        System.out.println("Truck started work");
+    }
+    @java.lang.Override
+    public void stopWork() {
+        System.out.println("Truck stopped work");
+    }

```

Comment:
```suggestion
    @Override
    public void doWork() {
        System.out.println("Truck started work");
    }
    
    @Override
    public void stopWork() {
        System.out.println("Truck stopped work");
    }
```

fix other classes accordingly

Reply:
Please check the changes made